### ISSUE TITLE
About datasets

### ISSUE THREAD

**Issue by striver123456** [awei669/VQ-Font#1]
Great Work!!!    How did you create the dataset?

**Comment by awei669** [comment]
Thank you for your interest in our work！

1. You should collect different ttf files, determine the source font, and divide the training set and test set.
2. Determine the generated character set (it also needs to be divided into training character set ‘train_unis’ and test character set ‘val_unis’), use font2image.py to generate images, see readme for organizational form.
3. Run script ./build_trainset.sh. 

Hope can help you～

**Comment by awei669** [comment]
If you have any further questions, re-open the issue.
